### PR TITLE
Feature 9

### PR DESCRIPTION
**Pull request by KooKouse**

### CODE REVIEW

File: cmd/quick-generator/main.go [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+func saveContentToFile(fileName string, content []byte) error {
+	once := sync.Once{}
+	once.Do(func() {
+		err := os.Mkdir("model", 0755)
+		if err != nil {
+			fmt.Printf("Error creating directory: %v\n", err)
+			return
+		}
+	})
```
Comment (HaHA3507) [inline cmd/quick-generator/main.go:146]: 好想没有判断文件夹是否已经存在的情况

File: cmd/quick-generator/main.go [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+	parser := NewParser(tableName)
+	for _, column := range createTableStmt.TableSpec.Columns {
+		c := &Column{
+			Name:  column.Name.String(),
+			Alias: column.Name.String(),
+			Type:  column.Type.Type,
+		}
+		if column.Type.KeyOpt == 1 {
+			c.IsPrimaryKey = true
+		}
```
Comment (HaHA3507) [inline cmd/quick-generator/main.go:118]: 并没有读出id是primary key
image

Comment (HaHA3507) [reply cmd/quick-generator/main.go:118]: CREATE TABLE `block` (
                         `id` int NOT NULL AUTO_INCREMENT,
                         `block_code` varchar(10) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci DEFAULT NULL,
                         `block_name` varchar(10) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci DEFAULT NULL,
                         `stock_code` varchar(10) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci DEFAULT NULL,
                         `stock_name` varchar(30) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci DEFAULT NULL,
                         `type` varchar(10) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci DEFAULT NULL,
                         PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=68329 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

Comment (KooKouse) [reply cmd/quick-generator/main.go:118]: 额外解析了Index的内容

**Review by HaHA3507 — APPROVED** [review]

**Pull request merged**